### PR TITLE
Transliteration of Technical Community Doc to Markdown

### PR DESCRIPTION
**Pull request by edwarnicke**
Original: https://fd.io/docs/tsc/FD.IO-Technical-Community-Document-12-12-2017.pdf

This is an attempt to simply transfer formats from PDF to Markdown.
It should not change the contents of the document at all.

Signed-off-by: Ed Warnicke <hagbard@gmail.com>

### CODE REVIEW

File: content/community/tscdocs/technical_community_doc.md [hunk 1]
```diff
@@ -0,0 +1,440 @@
...
+
+* Introduction
+* Requirements for Participation
+* The scheduling, practice, quality (but not technical content) requirements for 
+a project to participate.  In practice these are expected to build upon the 
+‘Mature’ project standards.
+* Milestones & Release Candidates
+* Dates and requirements for milestones and release candidates
+  * This should include by which Milestone a Project must opt in to join
+  *This should include by which Milestone freezes of various sorts (for example
```
Comment (mdr78) [inline content/community/tscdocs/technical_community_doc.md:426]: Need to fix bullet.

File: content/community/tscdocs/technical_community_doc.md [hunk 2]
```diff
@@ -0,0 +1,440 @@
...
+traceable decision making process.
+
+### 3.2.2 Committer Lifecycle
+
+#### 3.2.2.1 Adding Committers
+
+Initial Committers for a project will be specified at project creation.
+
+Committer rights for a project are earned via code contribution and community
+trust.   Committers for a project select and vote for new Committers for that project,
```
Comment (mdr78) [inline content/community/tscdocs/technical_community_doc.md:90]: Double space

File: content/community/tscdocs/technical_community_doc.md [hunk 3]
```diff
@@ -0,0 +1,440 @@
+# Technical Community Document
+
+**Relentlessly focused on data IO speed and efficiency for more flexible and
+scalable networks and storage**
+
+# Introduction
+
+This Technical Community Document sets forth additional details concerning the
+operation of the technical community of FD.IO Project a Series of LF Projects, LLC
+(the “Project”).  Capitalized terms not defined in the Technical Community
```
Comment (mdr78) [inline content/community/tscdocs/technical_community_doc.md:10]: Double space

File: content/community/tscdocs/technical_community_doc.md [hunk 4]
```diff
@@ -0,0 +1,440 @@
...
+Document will have the meanings ascribed to them in the technical charter for the
+Project.  The Technical Community Document may be amended from time to time by
```
Comment (mdr78) [inline content/community/tscdocs/technical_community_doc.md:12]: Double space

File: content/community/tscdocs/technical_community_doc.md [hunk 5]
```diff
@@ -0,0 +1,440 @@
...
+
+#### 3.2.3.2 Project Technical Leader Voters
+
+Only Committers for a project are eligible to vote for a project’s PTL.
+
+#### 3.2.3.3 Project Technical Leader Election Mechanics
+
+Election of a project’s PTL shall use a multiple-candidate method, e.g.:
+
+
```
Comment (mdr78) [inline content/community/tscdocs/technical_community_doc.md:133]: Double carriage return

File: content/community/tscdocs/technical_community_doc.md [hunk 6]
```diff
@@ -0,0 +1,440 @@
...
+* Terminate
+* Management Agents
+* Control/Manage IO/Processing
+* Supporting Projects
+* Testing/Tools/Infrastructure
+* Integration with other systems
+
+### 3.3.2 Project States
+
+
```
Comment (mdr78) [inline content/community/tscdocs/technical_community_doc.md:162]: Double carriage return

File: content/community/tscdocs/technical_community_doc.md [hunk 7]
```diff
@@ -0,0 +1,440 @@
...
+The TSC will elect from among TSC members, the TSC Chairperson.
+
+#### 4.2.2.1 TSC Chair Candidates
+
+Candidates for a TSC Chair must be TSC Members.
+
+Candidates must self nominate.
+
+
```
Comment (mdr78) [inline content/community/tscdocs/technical_community_doc.md:362]: Double carriage return

File: content/community/tscdocs/technical_community_doc.md [hunk 8]
```diff
@@ -0,0 +1,440 @@
...
+# 5 Evolution of Technical Community Document
+
+The TSC has ability to amend this Technical Community Document, subject to the
+technical charter of the Project.
+
+The normal amendment process is for either the TSC or the Committers-at-Large to
+propose changes using simple majority of the full TSC to resolve conflicts.  
+
+
```
Comment (mdr78) [inline content/community/tscdocs/technical_community_doc.md:440]: Double carriage return

**Review by mdr78 — CHANGES_REQUESTED** [review]
Good job Ed.

* 27 instances of double spacing - I annotated a few of these, but it is easier to do a find replace than me pointing out each one.
* I noted a few instances of double carriage return.
* There was one bullet formatting issue at the end of the document. 

Other than those, it is all good.

File: content/community/tscdocs/technical_community_doc.md (hunk 2 above)
Comment (edwarnicke) [reply content/community/tscdocs/technical_community_doc.md:90]: This is a triple space.  Fixed :)

File: content/community/tscdocs/technical_community_doc.md (hunk 1 above)
Comment (edwarnicke) [reply content/community/tscdocs/technical_community_doc.md]: Fixed

File: content/community/tscdocs/technical_community_doc.md (hunk 3 above)
Comment (edwarnicke) [reply content/community/tscdocs/technical_community_doc.md:10]: I still believe in the oxford comma and double spacing after a period :)

File: content/community/tscdocs/technical_community_doc.md (hunk 4 above)
Comment (edwarnicke) [reply content/community/tscdocs/technical_community_doc.md:12]: I still believe in the oxford comma and double spacing after a period :)

File: content/community/tscdocs/technical_community_doc.md (hunk 6 above)
Comment (edwarnicke) [reply content/community/tscdocs/technical_community_doc.md]: I still believe in the oxford comma and double spacing after a period :)

File: content/community/tscdocs/technical_community_doc.md (hunk 5 above)
Comment (edwarnicke) [reply content/community/tscdocs/technical_community_doc.md]: Fixed

File: content/community/tscdocs/technical_community_doc.md (hunk 8 above)
Comment (edwarnicke) [reply content/community/tscdocs/technical_community_doc.md]: Fixed

**Pull request merged**